src/dao.ts

```diff
@@ -142,6 +145,32 @@ export class DAO implements IStateful<IDAOState> {
   public ethBalance(): Observable<number> {
     return this.context.getBalance(this.address)
   }
+
+  public getContract(name: string) {
```

Comment:
I was more thinking of writing something liek this: https://github.com/daostack/client/issues/44 , where it makes sense to ask to the DAO: hey, what contracts are you using, and sos this would be the rigth place.

But given that the info now comes from migration.json, you are right, I'll move it